extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/impl/ChangeEventElementJsonImpl.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.cdc.impl;
+
+import com.fasterxml.jackson.jr.annotationsupport.JacksonAnnotationExtension;
+import com.fasterxml.jackson.jr.ob.JSON;
+import com.hazelcast.jet.cdc.ChangeEventElement;
+import com.hazelcast.jet.cdc.ParsingException;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+
+class ChangeEventElementJsonImpl implements ChangeEventElement, IdentifiedDataSerializable {
+
+    private static final JSON J = JSON.builder().register(JacksonAnnotationExtension.std).build();
+
+    private String json;
+    private Map<String, Object> content;
+
+    ChangeEventElementJsonImpl() { //needed for deserialization
+    }
+
+    ChangeEventElementJsonImpl(@Nonnull String json) {
+        this.json = Objects.requireNonNull(json);
+    }
+
+    @Override
+    @Nonnull
+    public <T> T asPojo(@Nonnull Class<T> clazz) throws ParsingException {
+        Objects.requireNonNull(clazz, "class");
+        try {
+            return J.beanFrom(clazz, json);
+        } catch (IOException e) {
+            throw new ParsingException(e.getMessage(), e);
```

Comment:
what's the point of rethrowing here? what's wrong with IOException?

Reply:
I use `ParsingException` to signal various failures in parsing, which in my mind is not I/O (failing to receive the JSON string from somewhere, that would be I/O, parsing it after it's in memory, that doesn't seem to be... arguable). 

Anyways, beyond my rumblings, `IOException` is too generic, even Jackson jr have their own `JsonProcessingException`. 

I can change it if you feel strongly about it, I don't mind, just wanted to clarify my reasoning.